packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -156,7 +156,7 @@ function persistedProfileMigrationToV9(existingProfile: IPersistedProfile): void
 }
 
 function persistedProfileMigrationToV10(existingProfile: IPersistedProfile): void {
-    const network = NETWORK?.[existingProfile?.networkProtocol]?.[existingProfile?.networkType]
+    const network = NETWORK_METADATA_MAP?.[existingProfile?.networkProtocol]?.[existingProfile?.networkType]
```

Comment:
we never need the `?` after a defined constant

Reply:
Yes true